numpy/lib/tests/test_twodim_base.py

```diff
@@ -273,6 +276,27 @@ def test_binparameter_combination(self):
         assert_array_equal(H, answer)
         assert_array_equal(xe, array([0., 0.25, 0.5, 0.75, 1]))
 
+    @requires_array_function
+    def test_dispatch(self):
+        class ShouldDispatch:
+            def __array_function__(self, function, types, args, kwargs):
+                return types, args, kwargs
```

Comment:
I'm not sure it's useful to return/check `args` and `kwargs` -- those are just directly passed on from the calling functions.

Reply:
Yes, you're right, but it went in already....  And I guess it doesn't hurt to check.